packages/node_modules/@webex/webex-core/src/lib/services/service-catalogs.js

```diff
@@ -98,6 +98,98 @@ export default class ServiceCatalogs {
       [...this.hosts];
   }
 
+  /**
+   * Filter the {@link ServiceCatalogs#hosts} array against their location in
+   * reference to the authenticated user.
+   *
+   * @private
+   * @memberof ServiceCatalogs
+   * @param {boolean} [local] - Filter for the host location.
+   * @returns {Array<ServiceHost>} - The filtered host array.
+   */
+  filterLocal(local) {
+    return (typeof local === 'boolean') ?
```

Comment:
Do we need this extra "protection"? I think filtering where we match `host.local === local` should be good enough.

Reply:
There are some cases in which we would like to retrieve remote hosts only [`local === false`]. The most common use case is:
- When retrieving a priority URL for a remote host.

When this value is set to `true`, it will return what we would typically want to retrieve when setting up most of our requests.

Alternatively, when attempting to validate a service exists at all, we would pass an `undefined` value into this method.